packages/TelegramClient-Core.package/TCCChatsHandler.class/instance/openNewChat..st

```diff
@@ -1,4 +1,9 @@
 events
 openNewChat: aChatEvent
 
-	self core chats triggerEvent: #openNewChat with: (TCCChat newFromChatEvent: aChatEvent).
\ No newline at end of file
+	| newChat |
+	newChat := TCCChat newFromChatEvent: aChatEvent.
+	(self core chats hasChat: newChat id)
+		ifFalse: [self core chats add: newChat.]
+		ifTrue: [newChat := self core chats getChat: newChat id.].
+	self core chats triggerEvent: #openNewChat with: newChat.
```

Comment:
`self core chats`: i think we already talked about putting `chats` on the `ChatHandler`, didn't we?

Reply:
Yep we did and I guess that will come with the next refactoring. I think it would be a bit too much for this one :)